### DISCUSSION TITLE
The future from this project?

### DISCUSSION THREAD

**Discussion by astrO1** [bcakmakoglu/vue-flow#1418] [Q&A] [answered]
I'm asking this question because i have an big application with many users and i build that with Angular. 

I'm tired of the boilerplate and i programed in Vue.js in the past and really liked. So i'm switching the tecnology.

My problem is if this project is new and i want to pick something that will take long time. Because my app will use the features from this package a lot.

This decision is very important, because i see the react flow and is old and has a large comunity.

I prefer Vue.js over React but business is business.

By the way, really thanks for this amazing project!

**Comment by bcakmakoglu** [comment, marked as answer]
Not sure what your question is since you didn't really ask any question in your post.
Whether the project will still be maintained in the future? Yes, it will be.
Is the project new? You can take your time, dig a little bit into the commit history and will find that this project has been going on for almost 4 years now.

**Reply by astrO1** [reply]
The question is the title, the future =)

**Reply by bcakmakoglu** [reply]
Yeah but what exactly where you wondering about in the context of "future", that's what my confusion was. I guess it was just about maintenance of the library. 😅 
[Maybe this post is also relevant to your question.](https://github.com/bcakmakoglu/vue-flow/discussions/906#discussioncomment-9407554)
